### ISSUE TITLE
False positive in Lint/RescueWithoutErrorClass

### ISSUE THREAD

**Issue by kyrylo** [bbatsov/rubocop#4768]
Since v0.50 my project gets false positives with the following code:

```rb
begin
  # ...
rescue => ex
  puts ex # or whatnot
end
```

Ruby Style Guide says this code is `good`: https://github.com/bbatsov/ruby-style-guide#no-blind-rescues

--------

## Expected behavior

No offences.

## Actual behavior

```
% rubocop lib/airbrake-ruby/notice.rb
Inspecting 1 file
W

Offenses:

lib/airbrake-ruby/notice.rb:201:7: W: Lint/RescueWithoutErrorClass: Avoid rescuing without specifying an error class. (https://github.com/bbatsov/ruby-style-guide#no-blind-rescues)
      rescue => ex
      ^^^^^^

1 file inspected, 1 offense detected
```

## Steps to reproduce the problem

Grab this file: https://github.com/airbrake/airbrake-ruby/blob/v2.3.2/lib/airbrake-ruby/notice.rb#L199-L205

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
% rubocop -V
0.50.0 (using Parser 2.4.0.0, running on ruby 2.4.0 x86_64-darwin16)
```

**Comment by Drenmi** [comment]
Cop is working as intended. RuboCop has moved from just enforcing the style guide to be more configurable. Feel free to disable it if you prefer to rescue errors blindly. 🙂 

(In this case the style guide should arguably also be changed.)
